source/frontend/src/features/mapSideBar/compensation/detail/CompensationRequisitionDetailContainer.tsx

```diff
@@ -83,7 +83,8 @@ export const CompensationRequisitionDetailContainer: React.FunctionComponent<
       loading={
         loading || loadingCompReqProperties || loadingCompReqAcqPayees || loadingCompReqLeasePayees
       }
-      file={file}
+      fileProduct={file.product}
```

Comment:
is file guaranteed to be valued here? maybe file?.product to be safe.

Reply:
good catch